Formula/coreutils.rb

```diff
@@ -60,6 +65,7 @@ def install
       --without-selinux
     ]
 
+    system "aclocal", "-Im4"
```

Comment:
Isn't the `m4` directory already searched by default? Also, please add a comment explaining that this line can be removed when the patch is no longer needed.

Reply:
no. it will fail with:

```
./configure: line 5083: gl_EARLY: command not found
./configure: line 5084: syntax error near unexpected token `auto-gpl-compat'
./configure: line 5084: `gl_SET_CRYPTO_CHECK_DEFAULT(auto-gpl-compat)'
make: *** [config.status] Error 2
```